### ISSUE TITLE
Unable to push Unity project to GitHub

### ISSUE THREAD

**Issue by taulantxhakli** [github-for-unity/Unity#1143]

**Comment by taulantxhakli** [comment]
Solved it a while back and forgot about this post.
If anyone sees this and wonders what is going on and what to do. For starters, this happens because the packets gets lost through the push due to too many files. If you are pushing small amount of files and the packets still get lost, that is due to your internet. To fix this for large files, as this one was roughly around 13GB for a Unity project, I just sent assets little by little then the project itself.

**Comment by akashmanjunath2505** [comment]
Same problem here

**Comment by RunoLight** [comment]
Can you specify the Unity version and steps to replicate the problem?

**Comment by taulantxhakli** [comment]
> Can you specify the Unity version and steps to replicate the problem?

Apologies for the late reply, I have got through this issue in a non-conventional way. 
Seeing as this was a Unity project and it had gigabytes worth of assets, I pushed those individually through, leaving 1-2 Gb worth of assets each time I pushed through Git. Once I did all of the assets, I pushed the rest of the project repository, which at that point it was below 2 gigabytes. That is one way of doing it. I tried multiple methods in the Git terminal, but none seem to work for me. If anyone gets stuck with some of the sort, this is an alternative.
